bin/gen-credits.dart

```diff
@@ -6,8 +6,11 @@ import 'package:http/http.dart';
 import 'package:html/parser.dart';
 import 'package:pub_api_client/pub_api_client.dart';
 import 'package:pubspec_parse/pubspec_parse.dart';
+import 'package:spotube/models/logger.dart';
 
 void main() async {
+  final logger = getLogger("");
```

Comment:
Logger can't run without path_provider & this file is a dart CLI file. So it's not supposed be using any logging framework